cla-backend-go/project/service.go

```diff
@@ -121,6 +127,23 @@ func (s service) GetProjectByName(projectName string) (*models.Project, error) {
 
 // DeleteProject service method
 func (s service) DeleteProject(projectID string) error {
+	//Unassign gerrit repositories
+	err := s.gerritService.UnassignProject(projectID)
+	if err != nil {
+		log.Warnf("Unable to unassign gerrit repositories for project : %s error : %v", projectID, err)
+		return err
+	}
+	githubErr := s.repoService.UnassignProject(projectID)
+	if githubErr != nil {
+		log.Warnf("Unable to unassign github repositories for project : %s error :%v ", projectID, githubErr)
+		return githubErr
+	}
+	//Unassign signatures
+	signatureErr := s.signatureService.RemoveProject(projectID)
```

Comment:
Here we are not deleting the signature record. Maybe rename to: Invalidate or InvalidateProjectRecords